draft-ietf-scitt-architecture.md

```diff
@@ -747,8 +753,8 @@ A Transparency Service MUST ensure that a Signed Statement is registered before
 
 A Transparency Service MAY accept a Signed Statement with content in its unprotected header, and MAY use values from that unprotected header during verification and registration policy evaluation.
 
-However, the unprotected header of all Signed Statements in the Append-only log MUST be empty.
-A Transparency Service MUST replace the unprotected header with an empty unprotected header before inclusion in the Append-only log.
+However, the unprotected header of all Signed Statements committed to the Verifiable Data Structure MUST be empty.
+A Transparency Service MUST replace the unprotected header with an empty unprotected header before committing to the Verifiable Data Structure.
```

Comment:
```suggestion
```

Reply:
Duplicate?

Reply:
ah yeah, but also, I was replacing "included" with committed elsewhere. In the context of a Statement Sequence though, I am fine with included